docs/source/tutorial/tutorial-mutations.md

```diff
@@ -130,7 +130,7 @@ This is helpful because the Swift code generation will now generate a method tha
 In the `Query Variables` section of GraphiQL, update variables to use `tripID` as the key, and remove the array brackets from around the identifier: 
```

Comment:
looks like this needs to update `tripID` as well

Reply:
Right! Good catch. I've pushed new changes